src/Components/gunwookim/SpanLink.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+import './SpanLink.scss';
```

Comment:
미리 약속이 되어 있는 경우 구분하기 좋게 각각 띄워놓기도 하는데 저희는 일단 붙여 쓰고 있어서 붙여주시면 좋을 것 같습니다
```suggestion
import React from 'react';
import { Link } from 'react-router-dom';
import './SpanLink.scss';
```

Reply:
네 알겠습니다